GVFS/GVFS.UnitTests.Windows/Windows/Upgrader/NewProductUpgradeTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using GVFS.Common;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Git;
+using GVFS.Common.NuGetUpgrade;
+using GVFS.Tests.Should;
+using GVFS.UnitTests.Mock.Common;
+using GVFS.UnitTests.Windows.Mock.Upgrader;
+using Moq;
+using NUnit.Framework;
+
+namespace GVFS.UnitTests.Windows.Windows.Upgrader
+{
+    public class NewProductUpgradeTests
+    {
+        private MockTracer tracer;
+        private MockLocalGVFSConfig gvfsConfig;

```

Comment:
Minor: Since ever test ends up creating its own `gvfsConfig` I'm not sure there's any value in having `gvfsConfig` as a member variable.